app/components/InventoryOverview/index.tsx

```diff
@@ -649,6 +883,174 @@ const InventoryOverview = ({ navigation }: any) => {
 };
 export default InventoryOverview;
 
+const CoordinateOverviewModal = ({
+  coordinateModalShow,
+  setCoordinateModalShow,
+  coordinateIndex,
+  protocol,
+  cdnUrl,
+  inventory,
+  isSampleTree,
+  initialState,
+  navigation,
+  scrollPosition,
+  layoutAboveMap,
+  customModalPosition,
+  setCustomModalPosition,
+}: any) => {
+  const [imageSource, setImageSource] = useState<any>();
+  const [marker, setMarker] = useState<any>();
+  const [scientificSpecies, setScientificSpecies] = useState<any>();
+  const [countryCode, setCountryCode] = useState<string | undefined>();
+  useEffect(() => {
+    if (coordinateIndex || coordinateIndex === 0) {
+      if (isSampleTree !== null && !isSampleTree) {
+        setMarker(inventory?.polygons[0].coordinates[coordinateIndex]);
+        initiateMarkerData(inventory?.polygons[0].coordinates[coordinateIndex]);
+      } else if (isSampleTree !== null && isSampleTree) {
+        setMarker(inventory?.sampleTrees[coordinateIndex - 1]);
+        initiateMarkerData(inventory?.sampleTrees[coordinateIndex - 1]);
+      }
+    }
+    getUserInformation().then((data) => {
+      setCountryCode(data.country);
+    });
+  }, [coordinateIndex, coordinateModalShow]);
+
+  const initiateMarkerData = (marker: any) => {
+    if (marker) {
+      if (marker.imageUrl) {
+        const imageURIPrefix = Platform.OS === 'android' ? 'file://' : '';
+        setImageSource({
+          uri: `${imageURIPrefix}${RNFS.DocumentDirectoryPath}/${marker.imageUrl}`,
+        });
+      } else if (marker.cdnImageUrl) {
+        setImageSource({
+          uri: `${protocol}://${cdnUrl}/media/cache/coordinate/thumb/${marker.cdnImageUrl}`,
+        });
+      } else {
+        setImageSource(map_img);
+      }
+      if (isSampleTree) {
+        getScientificSpeciesById(marker.specieId).then((specie: any) =>
+          setScientificSpecies(specie?.scientificName),
+        );
+      }
+    }
+  };
+  const specieHeight = nonISUCountries.includes(countryCode)
+    ? marker?.specieHeight * meterToFoot
+    : marker?.specieHeight;
+  const specieDiameter = nonISUCountries.includes(countryCode)
+    ? marker?.specieDiameter * cmToInch
+    : marker?.specieDiameter;
+
+  const heightUnit = nonISUCountries.includes(countryCode)
+    ? i18next.t('label.select_species_feet')
+    : 'm';
+  const diameterUnit = nonISUCountries.includes(countryCode)
+    ? i18next.t('label.select_species_inches')
+    : 'cm';
+  return (
+    <Modal visible={coordinateModalShow} transparent>
+      <TouchableOpacity
+        style={styles.outsideModalContainer}
```

Comment:
If this does not work, I suggest encapsulating the `Image` with `TouchableOpacity` to close the modal.